README.md

```diff
@@ -1,13 +1,50 @@
-# Slothstagram
+## Slothstagram
 
-# Setting up
+## Setting up
 
-- `git clone` the repo
-- Make virtual enviornment and with within this enviornment for all changes with `mkvirtualenv slothstagram`
-- NOTE: When changing to another project use `deactivate` to exit the virtual enviornment. Use `workon slothstagram` to return to this project's enviornment.
+- Clone this repository
+- Create python virtual environment and make all changes within this environment.
+
+_NOTE: When changing to another project use `deactivate` to exit the virtual environment and `workon slothstagram` to return to this project's environment._
+
+## Clone repository
+
+`git clone https://github.com/Cosmic-Binturongs/slothstagram.git`
+
+## Create virtual environment
+
+`mkvirtualenv slothstagram`
+
+## Project Dependencies
+
+<details>
+<summary>Click to view all dependencies</summary>
+<br>
+  <ul>
+    <li>asgiref==3.5.1</li>
+    <li>click==8.1.3</li>
+    <li>dj-database-url==0.5.0</li>
+    <li>Django==4.0.4</li>
+    <li>django-admin==2.0.1</li>
+    <li>django-cors-headers==3.11.0</li>
+    <li>django-excel-base==1.0.4</li>
+    <li>django-excel-response2==3.0.2</li>
+    <li>django-six==1.0.4</li>
+    <li>djangorestframework==3.13.1</li>
+    <li>gunicorn==20.1.0</li>
+    <li>psycopg2-binary==2.9.3</li>
+    <li>python-dotenv==0.20.0</li>
+    <li>pytz==2022.1</li>
+    <li>screen==1.0.1</li>
+    <li>sqlparse==0.4.2</li>
+    <li>whitenoise==6.0.0</li>
+    <li>xlwt==1.3.0</li>
```

Comment:
Rather than listing out every dependency and sub-dependency, let's list the main things we have added to this Python project. They are:

- Django
- Django Rest Framework
- Psycopg2-binary
- Whitenoise
- Dotenv
- Gunicorn
- DJ Database URL